packages/kotti-ui/source/kotti-button/KtButton.vue

```diff
@@ -82,6 +82,7 @@ export default defineComponent<KottiButton.PropsInternal>({
 	justify-content: center;
 
 	font-weight: 600;
+	vertical-align: middle;
```

Comment:
Since this was a pretty weird bug, I think it makes sense to add a small comment on why this is necessary